### PR TITLE
Core sideloading

### PR DESCRIPTION
**Pull request by fr500**
## Guidelines

1. Rebase before opening a pull request
2. If you are sending several unrelated fixes or features, use a branch and a separate pull request for each
3. If possible try squashing everything in a single commit. This is particularly beneficial in the case of feature merges since it allows easy bisecting when a problem arises

## Description

[Description of the pull request, detail any issues you are fixing or any features you are implementing]

## Related Issues

[Any issues this pull request may be addressing]

## Related Pull Requests

[Any other PRs from related repositories that might be needed for this pull request to work]

## Reviewers

[If possible @mention all the people that should review your pull request]

### CODE REVIEW

File: menu/cbs/menu_cbs_ok.c [hunk 1]
```diff
@@ -1285,6 +1294,45 @@ static int generic_action_ok(const char *path,
...
+            settings_t            *settings = config_get_ptr();
+            char destination_path[PATH_MAX_LENGTH];
+            FILE *src, *dst;
+            char buffer[100];
+            int numr, numw;
+
+            fill_pathname_join(destination_path, settings->paths.directory_libretro, path_basename(action_path), sizeof(destination_path));
+            src = fopen(action_path, "rb");
+
+            if (!src)
```
Comment (twinaphex) [inline menu/cbs/menu_cbs_ok.c]: Would it be possible to use the VFS wrappers here instead?

Comment (RobLoach) [reply menu/cbs/menu_cbs_ok.c]: [vfs_implementation](https://github.com/libretro/libretro-common/blob/4b165e1d5edea374d5a56f3d477041560afb695e/include/vfs/vfs_implementation.h)?

**Comment by fr500** [comment]
I see no reason to do that, vfs makes sense for stuff that will interact
with the core and user data.
This is completely frontend side and nothing external is gonna use it.

I'm ok with someone else doing that but I have no interest, I achieved what
I needed to achieve to help a person with testing his cores on android,
that's what I was set out to do.

**Comment by grant2258** [comment]
the menu isint showing up on android for some reason

**Pull request closed without merging**